src/Chainweb/Graph.hs

```diff
@@ -41,6 +41,7 @@ module Chainweb.Graph
 -- * Chain Graph
 
 , ChainGraph
+, _chainGraphGraph
```

Comment:
The constructors of the `ChainGraph` must not be exported. 

There is a lensy `Getter` `chainGraphGraph` that is exported. For a non-lensy version the constructor fields would have to be renamed and `_chainGraphGraph` would have to be exported as separately defined read-only function.

Reply:
Oh, I didn't notice that.  Fixed.